joomla/joomla-cms
# com_installer "Install languages" UX Improvements

#### Description

This PR does some UX improvements to com_installer -> Install Languages:
- Use Jlayout Search Tools
- Adds colors to translation versions (lower - orange : Same - green)
- Moves warning message to the version tooltip.
- Adds a new column language code (with ordering)
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/12923354/4e45ebcc-cf4d-11e5-8b78-72c5fb205d63.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12962445/45c64d2a-d03d-11e5-9c48-ec4ebfb60c9d.png)
#### How to test
1. Install PR with latest staging
2. Go to Extensions -> Manage -> Install languages
   (check before and after PR)
3. Check if every filter if working fine
#### Observations

All version coluns are orange because you are in version 3.5.0 and there is no translation yet for 3.5.0.

Suggestions and improvements are welcome.
